src/Simplex/Chat/Types.hs

```diff
@@ -129,13 +129,15 @@ data User = User
     sendRcptsContacts :: Bool,
     sendRcptsSmallGroups :: Bool,
     userMemberProfileUpdatedAt :: Maybe UTCTime,
-    uiThemes :: Maybe UIThemeEntityOverrides
+    uiThemes :: Maybe UIThemeEntityOverrides,
+    userSuperpeer :: Bool
```

Comment:
I don't think we need it in type - it's not really a "user" is it? or how do you plan to use it?

If the point of having it as a user to uniformly subscribe, then maybe, but then it's a separate query to get all users as opposed to other users for the UI.

Reply:
it's for uniform subscription, yes, maybe other reasons too (I checked through code when added it as field - subscription was a reason, don't remember if there were others)

there's probably more work to do to make client work as supeerpeer, was going to filter out users then, will add TODO for now. Or maybe I thought even not filtering it out and instead filter it out in UI